test/parallel/test-primordials-access.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+// Flags: --no-warnings --expose-internals --expose-primordials
+
+const common = require('../common');
+const assert = require('assert');
+const primordials = require('internal/primordials');
+
+primordials.Date = common.mustCall(function() {
```

Comment:
```suggestion
primordials.Date = common.mustCall(() => {
```

Reply:
I don't think that would work – `Date` needs to be constructable.